### DISCUSSION TITLE
Modern Search deep linking will not work :-/ (v4)

### DISCUSSION THREAD

**Discussion by jbmueller0584** [microsoft-search/pnp-modern-search#1517] [Q&A]
Hi dear community,

first of all Modern Search is amazing ❤ Thank you so much for that great solution 🙏🏼

I created a page with a searchbox webpart that passes the searchTerm to a connected search results webpart on the same page. The search results webpart is also connected to a search filter webpart (as refiners) on the same page.

What i am now trying to achieve is to be able to preselect filters by URL params like so

https://xxx.sharepoint.com/sites/modernsearchtest/SitePages/Searchresults.aspx?filters=[{"n":"FileType","t":["docx"]}]

No matter what i'm doing, it wont work. I tried many things e. g. to encode the string appended after ?filters........, tried different managed properties etc. But it wont work unfornately. 

It is well documented vor v3 and v4 (i am using v4):

v3 https://microsoft-search.github.io/pnp-modern-search/v3/search-parts/search-results/#filters-deep-links
v4 https://microsoft-search.github.io/pnp-modern-search/usage/search-filters/#filter-deep-linking

Please give me an idea of what i'm doing wrong or what is the secret to get it work.

Thank you in advance!

Cheers,
Joerg

**Comment by FranckyC** [comment]
Hi @jbmueller0584, actually th URL format you are using is simply wrong. As the documentaion mentions:

> We recommend to use the URL generated from filter values selection instead of crafting the URL manually.

It means you must first filter your results with the wanted value(s) and copy the generated URL as deep link. Hope it helps.

**Comment by jbmueller0584** [comment]
@FranckyC Thank you for your feedback. Unfortunately, when i do a filtering, the url wont change and nothing is appended... I am using Edge Chrome. What am i doing wrong?

Kind regards,
Joerg

**Reply by FranckyC** [reply]
Are you using the latest version https://github.com/microsoft-search/pnp-modern-search/releases/tag/4.4.1? I've just tried with this package and it works as expected.
